controllers/pvc.go

```diff
@@ -154,6 +180,7 @@ func (r *VolumeSnapshotBackupReconciler) BindPVCToDummyPod(log logr.Logger) (boo
 						{
 							Name:      "vol1",
 							MountPath: "/mnt/volume1",
+							ReadOnly:  true,
```

Comment:
@eemcmullan Do we need to mark this as ready only ? Any particular reason ?

Reply:
Cephfs need this, and it is possible that other provisioners need it as well. It should be okay since we do not need the clone to have write permissions. Wdyt?

Reply:
@shubham-pampattiwar Yes tested with aws csi and it worked